migrations/scripts/3_deploy_libs.js

```diff
@@ -0,0 +1,39 @@
+const realm = process.env.WITNET_EVM_REALM || "default"
+const addresses = require("../addresses")[realm]
+const settings = require("../settings")
+
+let Witnet, CBOR
+
+module.exports = function (deployer, network) {
+  try {
+    Witnet = artifacts.require(settings.artifacts[realm].Witnet || settings.artifacts.default.Witnet || "Witnet")
+    CBOR = artifacts.require(settings.artifacts[realm].CBOR || settings.artifacts.default.CBOR || "CBOR")
```

Comment:
```suggestion
    Witnet = artifacts.require(settings.artifacts[realm].Witnet ||"Witnet")
    CBOR = artifacts.require(settings.artifacts[realm].CBOR || "CBOR")
```

Reply:
As stated before, if `settings.artifacts[realm].CBOR` was null (with `realm !== "default")l, we still want to try and read default value from `settings.artifacts.defaults.CBOR`.